electrum/simple_config.py

```diff
@@ -991,6 +991,11 @@ def _default_swapserver_url(self) -> str:
         short_desc=lambda: _('Send change to Lightning'),
         long_desc=lambda: _('If possible, send the change of this transaction to your channels, with a submarine swap'),
     )
+    WALLET_SORT_TX_OUTPUTS = ConfigVar(
+        'sort_tx_outputs', default=True, type_=bool,
+        short_desc=lambda: _('Sort transaction outputs'),
+        long_desc=lambda: _('Provides protection against client fingerprinting, but smart-contract like protocols can depend on UTXO ordering'),
```

Comment:
This description is cryptic. If this is going to be exposed in the GUI, it should be improved.

Reply:
Done, thanks for the comment, the old text was really cryptic, I agree.

Could you please check the new one and give me ideas if you know how to make it better?